pvs/default/app.conf

```diff
@@ -0,0 +1,21 @@
+#
+# Splunk app configuration file
+#
+
+[install]
+is_configured = 1
+build = 20
+
+[ui]
+is_visible = 1
+label = Tenable NNM (previously PVS) for Splunk
+
+[launcher]
+author = Tenable, Inc.
+description = The Tenable Nessus Network Monitor™ (NNM™), previously Passive Vulnerability Scanner (PVS™), monitors network traffic at the packet layer to determine topology and identify services, security vulnerabilities, suspicious network relationships, and compliance violations.
+version = 2.0.0
+
+[package]
+id = pvs
```

Comment:
Should this ID change to nnm also?

Reply:
The app id must match the root folder of the app.